desktop/app/lib/keychain/index.js

```diff
@@ -1,26 +1,31 @@
-const keytar = require( 'keytar' );
+const { safeStorage } = require( 'electron' );
```

Comment:
This was introduced with Electron v15, and it uses the system's keychain to perform encrypt/decrypt actions. From what I've read, it is just as secure as using `keytar` because both use the system keychain to manage the key.